TYPO3.Flow/Tests/Functional/Persistence/Fixtures/Image.php

```diff
@@ -43,4 +49,27 @@ public function setData($data)
     {
         $this->data = $data;
     }
+
+    /**
+     * @return CleanupObject
+     */
+    public function getRelatedObject()
+    {
+        return $this->relatedObject;
+    }
+
+    /**
+     * @param CleanupObject $relatedObject
+     */
+    public function setRelatedObject($relatedObject)
```

Comment:
You could use a type hint here `setRelatedObject(CleanupObject $relatedObject = null)`